lib/extensions/_extensions.scss

```diff
@@ -1,2 +1,2 @@
-$govuk-extensions-url-context: "/extension-assets"; @import "/Users/jonhindle/dev/manage-a-sentence/allocations-prototype/node_modules/govuk-frontend/govuk/all.scss";
-@import "/Users/jonhindle/dev/manage-a-sentence/allocations-prototype/node_modules/@ministryofjustice/frontend/moj/all.scss";
\ No newline at end of file
+$govuk-extensions-url-context: "/extension-assets"; @import "/Users/liammcmurray/Sites/allocations-prototype/node_modules/govuk-frontend/govuk/all.scss";
```

Comment:
I'll change these to relative in the next commit 👍

Reply:
Thanks Jon - not sure how that got created as an absolute path!